marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/flow/stepWithQuality/runStepWithQuality.sjs

```diff
@@ -0,0 +1,29 @@
+const test = require("/test/test-helper.xqy");
+const flowUtils = require("/data-hub/5/impl/flow-utils.sjs");
+
+let assertions = [];
+
+const content = [

```

Comment:
Can you add a test with invalid value for quality as well ?

Reply:
I think that's good, and that test can determine whether or not the ML error is good enough.

Note that an erroneous value would be really unusual here. "quality" is not used much IMO, and when Pari does use it, we can typically count on her not shooting herself in the foot with an invalid value.